docs/packages/pkg/Fruit.rst

```diff
@@ -0,0 +1,33 @@
+.. spelling::
+
+    Google Fruit
+
+.. index:: testing ; Fruit
+
+.. _pkg.Fruit:
+
+Fruit
+=====
+
+-  `Official <https://github.com/google/fruit>`__
+-  `Hunterized <https://github.com/hunter-packages/fruit>`__
+-  `Dev branch <https://github.com/ruslo/hunter/tree/Fruit>`__
+-  `Versions <https://github.com/ruslo/hunter/blob/master/cmake/projects/Fruit/hunter.cmake>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/Fruit/CMakeLists.txt>`__
+-  `Added by Alexey Shevchenko <https://github.com/FelikZ>`__
+   (`pr-1527 <https://github.com/ruslo/hunter/pull/1527>`__)
+
+Note: boost disabled by default since it has issues on some platforms. Use `CMAKE_ARGS FRUIT_USES_BOOST=ON` to enable it.
```

Comment:
I guess you mean
```
``CMAKE_ARGS FRUIT_USES_BOOST=ON``
```
instead of
```
`CMAKE_ARGS FRUIT_USES_BOOST=ON`
```

You can check how it's rendered by building documentation:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#add-documentation

Reply:
Also I think it make sense either to mention only option:
```
Use ``FRUIT_USES_BOOST=ON`` to enable it.
```

or add the full CMake code:
```cmake
# config.cmake
hunter_config(
    Fruits
    VERSION ${HUNTER_Fruits_VERSION}
    CMAKE_ARGS FRUIT_USES_BOOST=ON
)
```